app/keycloak.js

```diff
@@ -3,22 +3,25 @@ import React, { useState, useMemo, useEffect, useCallback } from 'react';
 import { ReactKeycloakProvider } from '@react-keycloak/web';
 import { useSelector, useStore } from 'react-redux';
 import _ from 'lodash';
+import * as jose from 'jose';
 import * as ActionTypes from './constants/actionTypes';
 import { sync, async } from './actions';
 import api from '../lib/core/api';
 import { ipcRenderer } from 'electron';
+import rollbar from './utils/rollbar';
 
 export let keycloak = null;
 
 let _keycloakConfig = {};
 let refreshTimeout = null;
 
-export const setTokenRefresh = (keycloak) => {
+export const setTokenRefresh = (tokenParsed) => {
   if (refreshTimeout) {
     clearTimeout(refreshTimeout);
     refreshTimeout = null;
   }
-  var expiresIn = (keycloak.tokenParsed['exp'] - new Date().getTime() / 1000 + keycloak.timeSkew) * 1000;
+  let timeskew = keycloak?.timeSkew ?? 0;
```

Comment:
Did we not have an issue with using the nullish coalescing operator (??) in Uploader with Webpack/Babel? Or am I thinking of another operator?

Reply:
It's only an issue if the code that's using it doesn't get run through Babel. I think the issue we were having was that a dependency was using it and we generally exclude `/node_modules` from Babel loading.

Reply:
Ah yes, it was a dependency that was causing the issue. That makes sense.